src/Sfa.Das.Sas.Indexer.ApplicationServices/Shared/Settings/AppServiceSettings.cs

```diff
@@ -48,6 +48,7 @@ public AppServiceSettings(IProvideSettings settingsProvider)
         public string GitUsername => _settings.GetSetting("GitUsername");
 
         public string GitPassword => _settings.GetSetting("GitPassword");
+        public string GitAccessToken => _settings.GetSetting("GitAccessToken");
```

Comment:
has GitAccessToken  been added to the config. not seeing this variable in the current pipeline.